catalog/spatial/csw/spatial-csw-common/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/common/CswSourceConfiguration.java

```diff
@@ -92,7 +103,11 @@ public String getPassword() {
     }
 
     public void setPassword(String password) {
```

Comment:
Couldn't this whole method be crystallized down to...
```java
public void setPassword(String password) {
    if (!StringUtils.isEmpty(password)) {
        this.password = encryptionService.decryptValue(password);
    }
}
```
This also protects you from a null password passed in on an update.